gwpy/table/io/losc.py

```diff
@@ -18,7 +18,7 @@
 
 """Fetch and parse an event catalog from GWOSC.
 """
-
+import json
```

Comment:
Please remove this unused import

```suggestion
```

Reply:
Pytest keeps failing in test_table.py when it runs:

def test_fetch_open_data_kwargs(self)
and
def test_fetch_open_data(self)

Specifically, I cannot find a parameter to check for masking since none of the values are masked anymore. This means the following line fails.

assert isinstance(gw170818["snr_pycbc"], MaskedConstant)